### DISCUSSION TITLE
pagview动画出现花屏现象

### DISCUSSION THREAD

**Discussion by maleah714** [Tencent/libpag#1753] [Q&A] [answered]
【版本信息】
com.tencent.tav:libpag:4.2.41-noffavc

【平台信息】
车机android系统
Android版本：Android 11

【预期的表现】
动画可以正常播放

【实际的情况】
偶现情况下，启动时初次播放动画出现花屏
排查后发现的报错如下：
09-11 20:59:19.520681 1538 20716 W System.err: java. lang. RuntimeException: init decoder timeout. cost: 3390ms
09-11 20:59:19.520761 1538 20716 W System.err:
at org. libpag.a.a(SourceFile:8)
09-11 20:59:19.520797
1538 20716 W System.err
at org. libpag.usa. run(SourceFile: 5)
09-11 20:59:19.520835
1538 20716 W System.err:
at android.os Handler handleCallback(Handler.iava:938)
09-11 20:59:19.520867 1538 20716 W System.err:
at android.os.Handler.dispatchMessage (Handler. iava: 99)
09-11 20:59:19.520918 1538
5193 E tafx
: Failed to instantiate texture from proxv.
09-11 20:59:19.520957 1538 20716 W System.err:
at android.os. Looper. loop (Looper. iava: 223)
09-11 20:59:19.520994 1538 20716 W System.err:
at android.os.HandlerThread. run (HandlerThread. iava: 67)

【Demo及附件】
视频和系统日志如下：
https://share.weiyun.com/gl7F6W8F

**Comment by domchen** [comment, marked as answer]
根据这个提示看是硬件解码器初始化失败，可能车机没有配置硬解能力，检查一下是否用了noffavc版本，Android端要用标准版本才具有软件解码的兜底。noffavc版本是留给有的业务已经自带了ffmpeg想要外部注入使用的场景，无论如何Android端都要陪着一份软件解码的回退才行正常使用，硬解失败概率很高。
